api/src/org/apache/cloudstack/api/response/AutoScaleVmProfileResponse.java

```diff
@@ -75,6 +75,7 @@
     @Parameter(name = ApiConstants.CS_URL,
                type = CommandType.STRING,
                description = "the API URL including port of the CloudStack Management Server example: http://server.cloud.com:8080/client/api?")
+    // leaving cloud.com reference above as it serves only as an example
```

Comment:
ok, in that perspect you have a point absolutely. The sentence would need to be rephrased, then. It now spells "example: http://..."!
Ive been going through the, code reviewing every occurrence of cloud.com,.... back in june!!! just before leaving the project to do some paid work ;) (hope my boss is alright with this remark)